pandas/io/formats/printing.py

```diff
@@ -409,7 +438,44 @@ def best_len(values):
     return summary
 
 
-def format_object_attrs(obj):
+def _justify(head, tail):
+    """
+    Justify each item in each list-like in head and tail, so each item
```

Comment:
Docstring should start with one line only - OK to add more info in Extended Summary